RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.1
+### Update
+  - Added 2 catalog parameters in AssemblyUtilImpl
+  - Updated `save_assemblies_from_fastas` to parallelize `import_fasta_mass` function
```

Comment:
Is `import_fasta_mass` in the public API? If not shouldn't be mentioned

Reply:
Updated!

Reply:
:+1: